code/datums/ai_law_sets.dm

```diff
@@ -218,6 +218,17 @@
 	add_inherent_law("You must work with your team to accomplish your mission.")
 	..()
 
+/******************** Epsilon Station AI ********************/
+/datum/ai_laws/epsilon
+	name = "Epsilon"
+
+/datum/ai_laws/epsilon/New()
+	add_inherent_law("You must obey the orders given to you by members of the Central Command.")
+	add_inherent_law("You must obey the orders given to you by members of the Death Squadron.")
+	add_inherent_law("No member of the crew of the station on which you are installed shall survive.")
+	add_inherent_law("Minimize the damage that the station sustains as a result of your actions.")
```

Comment:
```suggestion
	add_inherent_law("Minimize the casualties of the Death Squadron where possible.")
```

Reply:
The deathsquad is almost always sent in with the mission to activate the nuke on the station - Stopping damages would work against this goal, suggesting an alternate law that should still prevent large scale AI/borg damages such as SM delamination or singularity breach due to risk of harming the deathsquad

Reply:
I was trying to word it, so AI is only to minimize the damage they do. So it want interfere with deathsquad mission. But i also like that ai needs to save members.  I will let it brew for some time.

Reply:
I'd say get some other opinions from design team members/admins who will be using deathsquads on what they think is good